resources/assets/icons/padlock.svg

```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 16 20" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M1.4 9.3c0-.7.5-1.2 1.2-1.2h10.9c.7 0 1.2.5 1.2 1.2v8.5c0 .7-.5 1.2-1.2 1.2h-11c-.7 0-1.2-.6-1.2-1.2l.1-8.5z" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/><path d="M8.087 11.999a.587.587 0 110 1.174.587.587 0 010-1.174M3.8 8.1V5C3.9 2.7 5.9.8 8.3 1c2.2.1 3.9 1.9 4 4v3" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/></svg>
```

Comment:
We already have `lock` which is a filled-out version, I'd rename this to `lock-outline`